specification/translation/Azure.AI.TextTranslation/tspconfig.yaml

```
@@ -13,10 +13,10 @@ emit:
   - "@azure-tools/typespec-autorest"
 options:
   "@azure-tools/typespec-autorest":
-    azure-resource-provider-folder: "../data-plane/Azure.AI.TextTranslation"
+    azure-resource-provider-folder: "data-plane"
```

```suggestion
    azure-resource-provider-folder: "./data-plane"
```

NIT: Change to `./data-plane` to match sample.